### ISSUE TITLE
Feature / add a number ID for each panel on GUI table.

### ISSUE THREAD

**Issue by noeldum** [openpnp/openpnp#1444]
# Feature Request
## Describe the Feature
It would be really convenient to have a number ID in front of each panel. Above a certain number of panels (say 5) it currently become difficult to keep track on the panel we are working on as all entry in the table look more or less the same. Having to count the panels after we have focused the mouse elsewhere isn't too smooth on user experience. Say I do something on panel 6 then click somewhere for whatever reason. Once I want to come back to my panel 6 I have to count from the top to make sure I am back on the one I was working on. Not a huge effort but not really smooth.

## How Will It Look?
Basically like the picture below.
![Screenshot_2022-07-15_20-43-56](https://user-images.githubusercontent.com/1875345/179301952-9ccfd0ac-9f4d-49b5-81a4-3419825722e1.png)

## Why Do We Need It?
1. Improved User Experience
2. Useful for everybody I would say.

**Comment by GeoffState** [comment]
I am currently working on a solution which is looking good. There is a coding assumption that causes bug #1470 that I will solve to deliver this feature request.

**Comment by GeoffState** [comment]
### Thoughts
Further to above; after I have duplicated the files and the user edits them as in #1470, what am I do do with the disparate versions of the original files when it comes time to leave the application and offer to save files? My solution would be to modify the name of the files to include the, what I will call, **Ref** and leave the original disk file untouched.  This would yield new files in the original folder, **y_1.board.xml, y_2.board.xml, y_3.board.xml,** and **y_4.board.xml**: **y.board.xml** would remain untouched. 

### Moving Forward
What am I to do with these boards in the **Boards** pane when you reload the renamed files in another session? How about I try to organise them onto their unique **Ref** line? That would align with any board labels you have added to the physical work space. It would also mean they retain their prior **Ref** and be saved, if necessary, with the names allocated earlier. My code doesn't care about missing line numbers, but should you load two **z.board.xml**s using Refs **_1** to **_4**, I might be in a quandary... in which case I could warn and renumber at your response. Of course **z_5** and **z_6** already run in the same job with the original **y.board.xml** boards would be no problem.

How does this sound please?

**Comment by tonyluken** [comment]
@GeoffState,

Can you hold off on this?  Look at [NewPanelization.pdf](https://github.com/openpnp/openpnp/files/9844727/NewPanelization.pdf) to see what I already have implemented.  No use in us both working this same issue.

**Comment by GeoffState** [comment]
Tony, that's impressive! I'll set mine aside. Your work will
probably fix #1470 too.

Why don't we assign the tasks on github? I wonder if that's a paying
feature? I have no experience of working in a team environment and see the
project with 88 issues and only one assigned. I wonder how I request
assignment? I will research today. I am currently working on 4; minus two
now, #1444 and #1470.

I may be able to help you with testing but I don't have PnP hardware.

Very Best Regards
Geoff

**Comment by vonnieda** [comment]
Hi Geoff,

We've never been very formal about it as there's typically been few contributors at once. I believe you should be able to assign yourself to let others know you are working on something. Let me know if that doesn't work.

Jason

**Comment by GeoffState** [comment]
OK
Best Regards
Geoff

**Comment by tonyluken** [comment]
@noeldum,

Can we close this issue as I believe it was addressed by [PR#1507](https://github.com/openpnp/openpnp/pull/1578)?

Tony

**Comment by GeoffState** [comment]
Yes Tony, thanks.

Best Regards
Geoff
